x-pack/plugins/triggers_actions_ui/public/application/components/rules_setting/rules_settings_modal.tsx

```diff
@@ -165,10 +74,18 @@ export const RulesSettingsModal = memo((props: RulesSettingsModalProps) => {
     application: { capabilities },
   } = useKibana().services;
   const {
-    rulesSettings: { show, save, writeFlappingSettingsUI, readFlappingSettingsUI },
+    rulesSettings: {
```

Comment:
When I have existing rule settings with flapping set and switch to this branch, I don't see the query delay section show up in the rule settings.

Reply:
Resolved in this commit [dc7bbc8](https://github.com/elastic/kibana/pull/167433/commits/dc7bbc80b23f7dcd43d391cdd73844daeee0abb9)